spec/facades/quote_facade_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe QuoteFacade do 
+  describe 'class methods' do 
```

Comment:
I know it can feel redundant to the Service spec but I would also include the total count in here as well.